<?xml version="1.0"?>
<launch>

    <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan"
          respawn="true">

        <remap from="cloud_in" to="cloud_in_filtered"/>
        <!--        <remap from="scan" to="scan"/>-->
        <rosparam>
            target_frame: bot00_analyt # Leave disabled to output scan in pointcloud frame
            transform_tolerance: 1
            min_height: 0.0
            max_height: 0.0

            angle_min: -3.14 # -M_PI/2
            angle_max: 3.14 # M_PI/2
            angle_increment: 0.0174533 # M_PI/360.0
            scan_time: 6.1
            range_min: 0.0
            range_max: 400.0
            use_inf: true
            inf_epsilon: 1.0

            # Concurrency level, affects number of pointclouds queued for processing and number of threads used
            # 0 : Detect number of cores
            # 1 : Single threaded
            # 2->inf : Parallelism level
            concurrency_level: 1
        </rosparam>
    </node>
</launch>
